tests/foreman/api/test_host.py

```diff
@@ -391,45 +467,59 @@ def test_positive_create_with_compresource(self):
         host = entities.Host(
             compute_resource=compresource, location=loc, organization=org
         ).create()
-        self.assertEqual(host.compute_resource.read().name, compresource.name)
+        assert host.compute_resource.read().name == compresource.name
+        new_compresource = entities.LibvirtComputeResource(
+            location=[host.location], organization=[host.organization]
+        ).create()
+        host.compute_resource = new_compresource
+        host = host.update(['compute_resource'])
+        assert host.compute_resource.read().name == new_compresource.name
 
     @tier2
-    def test_positive_create_with_model(self):
-        """Create a host with model specified
+    def test_positive_create_and_update_with_model(self):
+        """Create and update a host with model specified
 
         :id: 7a912a19-71e4-4843-87fd-bab98c156f4a
 
-        :expectedresults: A host is created with expected model assigned
+        :expectedresults: A host is created and updated with expected model assigned
 
         :CaseLevel: Integration
         """
         model = entities.Model().create()
         host = entities.Host(model=model).create()
-        self.assertEqual(host.model.read().name, model.name)
+        assert host.model.read().name == model.name
+        new_model = entities.Model().create()
+        host.model = new_model
+        host = host.update(['model'])
+        assert host.model.read().name == new_model.name
 
     @tier2
-    def test_positive_create_with_user(self):
-        """Create a host with user specified
+    def test_positive_create_and_update_with_user(self, module_org, module_location):
+        """Create and update host with user specified
 
         :id: 72e20f8f-17dc-4e38-8ac1-d08df8758f56
 
-        :expectedresults: A host is created with expected user assigned
+        :expectedresults: A host is created and updated with expected user assigned
 
         :CaseLevel: Integration
         """
-        user = entities.User(organization=[self.org], location=[self.loc]).create()
+        user = entities.User(organization=[module_org], location=[module_location]).create()
```

Comment:
check if same available in global fixtures else worth creating there ?

Reply:
DONE.